src/EditorFeatures/Text/Extensions.cs

```diff
@@ -73,8 +73,18 @@ public static IEnumerable<Document> GetRelatedDocumentsWithChanges(this ITextSna
         /// associated with the buffer if it is linked into multiple projects or is part of a Shared Project. In this case, the <see cref="Workspace"/>
         /// is responsible for keeping track of which of these <see cref="Document"/>s is in the current project context.
         /// </summary>
+        [Obsolete("Use GetDocument instead.", error: true)]
```

Comment:
:exclamation: There is no need to set `error: true` here.

1. The method cannot ever be removed; it shipped as a public API
2. The behavior isn't changing, so calling it will continue to have the same effect it always did

:bulb: It would make sense to apply `[EditorBrowsable]` here to prevent the method from appearing in completion.

```suggestion
        [Obsolete("Use GetDocument instead.")]
        [EditorBrowsable(EditorBrowsableState.Never)]
```